jobs/JEVS_MESOSCALE_STATS

```diff
@@ -105,8 +105,14 @@ else
   exit
 fi
 
-# Execute the Script
-$HOMEevs/scripts/${COMPONENT}/${STEP}/exevs_${COMPONENT}_${MODELNAME}_${VERIF_CASE}_${STEP}.sh
+
+# Execute the script.
+if [ $MODELNAME = sref ] ; then
+   $HOMEevs/scripts/${STEP}/${COMPONENT}/exevs_${COMPONENT}_${MODELNAME}_${VERIF_CASE}_${STEP}.sh
+else
+   $HOMEevs/scripts/${STEP}/${COMPONENT}/exevs_${COMPONENT}_${MODELNAME}_${VERIF_CASE}_${STEP}.sh
+fi
```

Comment:
Is this if block needed?  It looks like both parts of the if and else are running the same script ... ?

Reply:
This block was used to separate sref in the last PR, but now we don't need it, and removed here.